sensor/Dockerfile.raspberrypi4-64

```diff
@@ -0,0 +1,25 @@
+FROM balenalib/raspberrypi3-python:3-build
+
+ARG BSEC_FILENAME=BSEC_1.4.7.4_Generic_Release.zip
+
+RUN install_packages \
+        unzip
+
+WORKDIR /usr/src/app
+RUN git clone https://github.com/balena-io-playground/bsec_bme680_linux.git
```

Comment:
the `make` is done in a further step down, right? I would bring them closer together, and use a single `RUN` that only keeps what it needs (ie: clone - make - make install or similar - remove sourcecode not required), which usually shrinks the source considerably.

Reply:
I'll take a look at improving this in a separate PR (and also looking at compiling for aarch64 at the same time).